### ISSUE TITLE
A few mixin errors

### ISSUE THREAD

**Issue by jeremyzilar** [uswds/uswds#3011]
There are a few issues I am getting with some of the mixins in `2.0`

### Outline

**When I use** `@include u-outline('05');`
I am getting the error — 
```
`05` is not a valid outline value.
```

**When I use** `@include u-outline('1px');`
I am getting the error — 
```
`1px` is not a valid outline value.
```

**When I use** `@include u-outline(1);`
I am getting the error — 
```
1 is not a valid outline width. Accepted values: 1px, 2px, 0, 05
```

---

### Overflow

When I use `@include u-overflow-x('hidden');`
I am getting the error — 
```
o mixin named u-overflow-x
```

**Comment by thisisdano** [comment]
I can fix most of the errors here, but `outline` currently only accepts the values listed — do you need it to accept 1 unit as well?

**Comment by maya** [comment]
Fixed in #3012.
